obulaworld/population-management
# Setup the server and routes

### What does this PR do?
This PR sets up the server and routes 👇🏽

### Description of Task to be completed?
* setup the server with express
* setup the routes
* structure the project and organize folders and files

### How should this be manually tested?
- git clone this app - git clone https://github.com/obulaworld/population-management.git
- navigate to the cloned app
- git checkout to this branch - git checkout -b ch-setup-server
- set up the .env with the .env.sample file
- install dependencies - npm install
- start the app - npm run server
- navigate to localhost:port/api/v1 on the browser
- you should see a welcome message

### Any background context you want to provide?
N/A

### What are the relevant pivotal tracker stories?
N/A

### Screenshots (if appropriate)
N/A

### Questions:
N/A
